proxy/secret_test.go

```diff
@@ -174,8 +179,9 @@ func Test_CreateSecret_Conflict409(t *testing.T) {
 	s := test.MockHttpServerStatus(t, http.StatusConflict)
 
 	secret := types.Secret{
-		Name:  "secret-name",
-		Value: "secret-value",
+		Name:      "secret-name",
```

Comment:
What is the error code returned when the namespace is not allowed? if it 409, then everything looks good, but if it is a new status, then we should probably have a test to ensure we are handling that status code correctly

Reply:
Yes, please could you do a quick test to show what happens with `-n kube-system` @viveksyngh ?

Reply:
Or any namespace without the `openfaas` annotation, actually.

Reply:
Since it is a mock, the value isn't all that special, the mock just needs to return the status code that the gateway would prodvide

Reply:
the status in case of non `kube-system` namespace in 400. We should also handle that inside proxy functions and print the error messages.

Reply:
Adding only unit tests would not be sufficient I guess. Let me know what you all think ?